src/ProjectContents/useContentsStore.tsx

```diff
@@ -40,16 +48,27 @@ export const useContentsStore = create<ContentsStore>((set, get) => ({
   ...initialState,
 
   actions: {
-    setContents: ({ name, content, schema }) => {
-      const jsonSchema = { ...schema };
+    setSchema: schema => {
+      set({ currentSchema: schema });
+    },
+    setContent: content => {
+      const copyContent = { ...content };
+      delete copyContent._createdAt;
+      delete copyContent._updatedAt;
+      delete copyContent._id;
+
+      set({ content: copyContent });
```

Comment:
```suggestion
      set({ content: omit(content, ["_createAt","_updatedAt","_id"]) });
```
이 로직에서는 불변성이 지켜지고 있기는 하지만, 불변성을 지키기에 delete를 사용하는 건 좋지 않을 것 같아요...! 혹시나 실수할 여지가 있어서요~

Reply:
좋네요~ 반영하겠습니다!